src/net.h

```diff
@@ -188,6 +180,31 @@ enum class ConnectionType {
      * AddrMan is empty.
      */
     ADDR_FETCH,
+
+    /**
+     * Manual-block-relay connections are manually added nodes (via -addnode)
+     * that behave like BLOCK_RELAY connections in that they don't participate in
+     * transaction and address relay, and also share the properties of MANUAL
+     * connections. When connected to a privacy network such as
+     * Tor, manual block-relay-only connections to IPv4 or IPv6 nodes can be
+     * used for hardening against eclipse attacks, while at the same time
+     * making fingerprinting attacks more difficult and preserving a higher degree
+     * of privacy by not transmitting transactions and addresses over these links.
+     */
+    MANUAL_BLOCK_RELAY,
+};
+
+struct AddedNodeInfo {
+    std::string strAddedNode;
+    CService resolvedAddress;
+    bool fConnected;
+    bool fInbound;
+    ConnectionType conn_type;
+};
+
+struct AddedNodeEntry {
+    std::string m_node_address;
+    ConnectionType conn_type;
```

Comment:
Be consistent with the `m_` prefix (either `m_node_address` and `m_conn_type` or `node_address` and `conn_type`). I prefer naming all members of structs with the `m_` prefix, but opinions vary.

Reply:
changed to `m_conn_type`